gcloud-java-storage/src/test/java/com/google/cloud/storage/BucketTest.java

```diff
@@ -249,33 +247,35 @@ public void testGet() throws Exception {
   }
 
   @Test
-  public void testGetAll() throws Exception {
+  public void testGetAllArray() throws Exception {
     initializeExpectedBucket(4);
-    Capture<BatchRequest> capturedBatchRequest = Capture.newInstance();
-    List<Result<Blob>> batchResultList = new LinkedList<>();
-    for (Blob info : blobResults) {
-      batchResultList.add(new Result<>(info));
-    }
-    BatchResponse response = new BatchResponse(Collections.<Result<Boolean>>emptyList(),
-        Collections.<Result<Blob>>emptyList(), batchResultList);
     expect(storage.options()).andReturn(mockOptions);
-    expect(storage.submit(capture(capturedBatchRequest))).andReturn(response);
-    expect(storage.options()).andReturn(mockOptions).times(3);
-    replay(storage);
+    List<BlobId> blobIds = Lists.transform(blobResults, new Function<Blob, BlobId>() {
+      @Override
+      public BlobId apply(Blob blob) {
+        return blob.blobId();
+      }
+    });
+    EasyMock.expect(storage.get(blobIds)).andReturn(blobResults);
+    EasyMock.replay(storage);
     initializeBucket();
-    List<Blob> blobs = bucket.get("n1", "n2", "n3");
-    Set<BlobId> blobInfoSet = capturedBatchRequest.getValue().toGet().keySet();
-    assertEquals(batchResultList.size(), blobInfoSet.size());
-    for (BlobInfo info : blobResults) {
-      assertTrue(blobInfoSet.contains(info.blobId()));
-    }
-    Iterator<Blob> blobIterator = blobs.iterator();
-    Iterator<Result<Blob>> batchResultIterator = response.gets().iterator();
-    while (batchResultIterator.hasNext() && blobIterator.hasNext()) {
-      assertEquals(batchResultIterator.next().get(), blobIterator.next());
-    }
-    assertFalse(batchResultIterator.hasNext());
-    assertFalse(blobIterator.hasNext());
+    assertEquals(blobResults, bucket.get("n1", "n2", "n3"));
+  }
+
+  @Test
+  public void testGetAllIterable() throws Exception {
+    initializeExpectedBucket(4);
+    expect(storage.options()).andReturn(mockOptions);
+    List<BlobId> blobIds = Lists.transform(blobResults, new Function<Blob, BlobId>() {
+      @Override
+      public BlobId apply(Blob blob) {
+        return blob.blobId();
+      }
+    });
+    EasyMock.expect(storage.get(blobIds)).andReturn(blobResults);
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Unnecessary use of fully qualified name 'EasyMock.expect' due to existing static import 'org.easymock.EasyMock.expect'](https://www.codacy.com/app/mziccard/gcloud-java/file/2448215795/issues/source?bid=3123103&fileBranchId=3247265#l275)